Lib/test/test_asyncio/test_taskgroups.py

```diff
@@ -190,7 +190,11 @@ async def runner():
         await asyncio.sleep(0.1)
 
         self.assertFalse(r.done())
-        r.cancel("test")
+        with self.assertWarnsRegex(
+            DeprecationWarning,
+            "Passing 'msg' argument"
+        ):
+            r.cancel("test")
```

Comment:
I would just remove the “test” arg to cancel(), here and below in this file. It serves no real purpose.

Reply:
Done